src/Traits/HelperTestCaseTrait.php

```diff
@@ -105,4 +105,29 @@ public static function assertMethodExists($method, $class, $msg = '')
 
         static::assertTrue(method_exists($class, $method), $msg);
     }
+
+    /**
+     * Assert that the element exists in the json.
+     *
+     * @param mixed  $needle
+     * @param array  $haystack
+     * @param string $msg
+     *
+     * @return void
+     */
+    public static function assertInJson($needle, $haystack, $msg = '')
+    {
+        if ($msg === '') {
+            $msg = "Expected the array to contain the element '$needle'.";
+        }
+
+        $array = json_decode($needle, true);
+
+        if (json_last_error() !== JSON_ERROR_NONE) {
```

Comment:
Maybe just throw a plain `Exception` with a useful message. It would be incorrect to fail the assertion you see. A more correct way is to throw an exception to say the input wasn't valid. Maybe even an "InvalidArgument" exception would be better?